scipy/stats/tests/test_stats.py

```diff
@@ -4480,6 +4480,35 @@ def test_1d_list_neg(self):
         with np.errstate(invalid='ignore'):
             check_equal_gmean(a, desired)
 
+    def test_weights_1d_list(self):
```

Comment:
One might say that these should be `@parametrize`d, but I think they're already pretty short, and it's nice that each one has a meaningful name.

Reply:
> One might say that these should be `@parametrize`d, but I think they're already pretty short, and it's nice that each one has a meaningful name.

I'd think an string that is unused in the test could be passed as argument w/the name/context given in the string. 

Yeah these seem ok w/o the parametrize for now.